sources/us/in/hamilton.json

```diff
@@ -9,8 +9,9 @@
         "state": "in",
         "county": "Hamilton"
     },
-    "data": "http://gis.hamiltoncounty.in.gov/ArcGIS/rest/services/E911/MapServer/0",
-    "protocol": "ESRI",
+    "data": "ftp://ftp.hamiltoncounty.in.gov/GIS/Data/Address Points/E911_Addresses_web.zip",
```

Comment:
Based on https://results.openaddresses.io/sources/us/in/hamilton the previous source appears to be stale. This file however looks to be updated continuously (it was last updated today).

The new source has `1021 Domino Dr` and the old source did not (this is the reason that I looked into this)